main/webapp/modules/core/langs/translation-es.json

```diff
@@ -308,10 +308,10 @@
 		"comma-sep": "Delimitado por comas",
 		"html-table": "Tabala HTML",
 		"excel": "Excel (.xls)",
-		"excel-xml": "Excel 2007+ (.xlsx)",
+		"excel-xml": "Excel en XML (.xlsx)",
 		"odf": "Hoja de cálculo ODF",
-		"triple-loader": "Triple loader",
-		"mqlwrite": "MQLWrite",
+		"triple-loader": "Triple loader (Firebase)",
```

Comment:
did you meant Freebase instead of Firebase. 
MQL and triple loader aren't specific to freebase.